app/templates/board.html

```diff
@@ -2,35 +2,68 @@
-                <span class="bz-item-id">#{{ item.id }}</span>
-            </a>
+            <div ng-hide="ColumnsScroll" style="height:200px; solid #ccc;font:16px/26px Georgia, Garamond, Serif;overflow:auto;">
+                <a href="show_bug.cgi?id={{ item.id }}" class="list-group-item" ng-repeat="item in bug_list | filter: { status: board }">
+                    <p class="list-group-item-text bz-item-desc">{{ item.desc }}</p>
+                    <span class="label label-default">{{ item.status }}</span>
+                    <span class="label label-primary" style="background-color:{{ item.product_color }}">{{ item.product }}</span>
+                    <span class="label label-info">{{ item.component }}</span>
+                    <span class="label label-default">{{ item.assignee }}</span>
+                    <small>{{ item.changed }}</small>
+                    <span class="bz-item-id">#{{ item.id }}</span>
+                </a>
+            </div>
+            <div ng-show="ColumnsScroll" >
+                <a href="show_bug.cgi?id={{ item.id }}" class="list-group-item" ng-repeat="item in bug_list | filter: { status: board }">
+                    <p class="list-group-item-text bz-item-desc">{{ item.desc }}</p>
+                    <span class="label label-default">{{ item.status }}</span>
+                    <span class="label label-primary" style="background-color:{{ item.product_color }}">{{ item.product }}</span>
+                    <span class="label label-info">{{ item.component }}</span>
+                    <span class="label label-default">{{ item.assignee }}</span>
+                    <small>{{ item.changed }}</small>
+                    <span class="bz-item-id">#{{ item.id }}</span>
+                </a>
+            </div>
         </div>
     </div>
     <!-- Assignee -->
     <div class="col-sm-{{ (12/board_sections_assignee.length) | number:0 }} bz-column" ng-repeat="board in board_sections_assignee" ng-show="ShowByAssignee">
         <button ng-click="showDiv = !showDiv" class = "btn btn-primary">{{ board }}</button>
         <div class="list-group" ng-hide="showDiv">
-            <a href="show_bug.cgi?id={{ item.id }}" class="list-group-item" ng-repeat="item in bug_list | filter: { assignee: board }">
-                <p class="list-group-item-text bz-item-desc">{{ item.desc }}</p>
-                <span class="label label-default">{{ item.status }}</span>
-                <span class="label label-primary" style="background-color:{{ item.product_color }}">{{ item.product }}</span>
-                <span class="label label-info">{{ item.component }}</span>
-                <span class="label label-default">{{ item.assignee }}</span>
-                <small>{{ item.changed }}</small>
-                <span class="bz-item-id">#{{ item.id }}</span>
-            </a>
+            <div ng-hide="ColumnsScroll" style="height:200px; solid #ccc;font:16px/26px Georgia, Garamond, Serif;overflow:auto;">
```

Comment:
Same applies here.

If this div is removed, we can toggle on/off a class (see above) which gives `list-group` scrollability without duplicate code.